packages/labs/src/components/sampleComponent/sampleComponent.tsx

```diff
@@ -0,0 +1,33 @@
+/* !
+ * (c) Copyright 2025 Palantir Technologies Inc. All rights reserved.
+ */
+
+import classNames from "classnames";
+import { createElement } from "react";
+
+import { SAMPLE_COMPONENT } from "../../common/classes";
+import { DISPLAYNAME_PREFIX, type Props } from "../../common/props";
+
+export interface SampleComponentProps extends Props, React.HTMLAttributes<HTMLElement> {
+    /**
+     * Name to display.
+     */
+    name: string;
+
+    /**
+     * HTML tag to use for element.
+     *
+     * @default "h1"
+     */
+    tagName?: keyof React.JSX.IntrinsicElements;
+}
+
+export const SampleComponent: React.FC<SampleComponentProps> = ({ name, className, tagName = "h1", ...htmlProps }) => {
```

Comment:
Is this a placeholder component? I'm assuming we replace this once we get our first labs component in?